heat/core/logical.py

```diff
@@ -307,3 +314,55 @@ def logical_xor(t1, t2):
     tensor([ False, False,  True])
     """
     return operations.__binary_op(torch.logical_xor, t1, t2)
+
+
+def _sanitize_close_input(x, y):
```

Comment:
2 underscores at the start of internal functions

Reply:
Thanks, done.